### PR TITLE
Protein function pipeline - update dbNSFP version 4.3a and SIFT 6.2.1

### PR DESCRIPTION
**Pull request by dglemos**
Update protein function pipeline to support dbSNFP 4.3a and the SIFT version 6.2.1 on codon.

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Pipeline/ProteinFunction/ProteinFunction_conf.pm [hunk 1]
```diff
@@ -185,12 +185,12 @@ sub default_options {
         dbnsfp_max_workers      => 50,
         dbnsfp_working          => $self->o('species_dir').'/dbnsfp_working',
         dbnsfp_annotation       => { GRCh37 =>
-                                      { file => $self->o('variation_data') . '/dbNSFP/4.2a/dbNSFP4.2a_grch37.gz',
-                                        version => '4.2a',
+                                      { file => $self->o('variation_data') . '/dbNSFP/4.3a/dbNSFP4.3a_grch37.gz',
```
Comment (likhitha-surapaneni) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ProteinFunction/ProteinFunction_conf.pm:188]: variation_data needs to be changed to point to the new path

**Review by likhitha-surapaneni — COMMENTED** [review]
lsf queue name to be changes to production instead of production-rh74

**Review by likhitha-surapaneni — COMMENTED** [review]
Path to accompanying BAM for include RefSeq transcripts to be changed to point to the new path

**Comment by diegomscoelho** [comment]
I think you can just update `pph_dir` (https://github.com/Ensembl/ensembl-variation/blob/732c35e7222ab0eaeaef91e3c1e6648ae25bf7b4/modules/Bio/EnsEMBL/Variation/Pipeline/ProteinFunction/ProteinFunction_conf.pm#L124)

**Comment by dglemos** [comment]
@diegomscoelho PolyPhen also needs to be updated in RunPolyPhen.pm and RunWeka.pm. I can do that separately.

**Review by diegomscoelho — APPROVED** [review]
LOTM! Changes work for SIFT and dbNSFP.

**Comment by likhitha-surapaneni** [comment]
Looks good to me

**Review by likhitha-surapaneni — APPROVED** [review]

**Pull request merged**